src/ChargeDriftModels/ADL/ADL.jl

```diff
@@ -116,18 +151,40 @@ function ADLChargeDriftModel(configfilename::Union{Missing, AbstractString} = mi
     h111 = VelocityParameters{T}(h111mu0, h111beta, h111E0, 0)
     electrons = CarrierParameters{T}(e100, e111)
     holes     = CarrierParameters{T}(h100, h111)
+    
+    
+    if "material" in keys(config)
+        config_material::String = config["material"]
+        if config_material == "HPGe"
+            material = HPGe
+        elseif config_material == "Si"
+            material = Si
+        else
+            @warn "Material \"$(config_material)\" not supported for ADLChargeDriftModel.\nSupported materials are \"Si\" and \"HPGe\".\nUsing \"$(Symbol(material))\" as default."
+        end
+    end
 
-    if "masses" in keys(config["drift"])
+    parameters = if "masses" in keys(config["drift"])
         ml = T(config["drift"]["masses"]["ml"])
         mt = T(config["drift"]["masses"]["mt"])
-        masses = MassParameters{T}(ml, mt)
+        ADLParameters{T}(ml, mt, material)
     else
-        masses = MassParameters{T}(T(1.64),T(0.0819))
+        ml = T(material_properties[Symbol(material)].ml)
+        mt = T(material_properties[Symbol(material)].mt)
+        ADLParameters{T}(ml, mt, material)
     end
 
-    ismissing(phi110) ? phi110 = T(config["phi110"]) : phi110 = T(phi110)  #if you give the angle of the 110 axis it will be used, otherwise read from config file
+    crystal_orientation::SMatrix{3,3,T,9} = if ismissing(phi110) 
+        if "phi110" in keys(config)
+            RotZ{T}(- π/4 - config["phi110"] )
+        else
+            transpose(parse_rotation_matrix(T, config, u"rad")) # replace u"rad" with the units in the config file
```

Comment:
The rotation matrix is transposed here. This is because the rotation of the coordinate system is a passive rotation.

In active rotations, an object is rotated around an axis. In passive rotations, the coordinate system is rotated such that all stationary objects are effectively rotated in the opposite direction with respect to the coordinate system.

However, transposing the matrix results in the fact that if the coordinate system should be first rotated by an angle `x` around the `X` axis and subsequently by an angle `z` around the `Z` axis, the corresponding implementation would have to look like this:
```yaml
"ZX": [z, x],
```
which is not intuitive. For this, we would need proper documentation.